media-sound/amarok/amarok-9999.ebuild

```diff
@@ -111,11 +114,39 @@ pkg_postinst() {
 		echo "${1} ($(has_version ${1} || echo "not ")installed)"
 	}
 
+	optfeature "Audio CD support" kde-apps/audiocd-kio
+
 	if [[ -z "${REPLACING_VERSIONS}" ]]; then
-		elog "You'll have to configure amarok to use an external db server:"
-		use mariadb && elog "    $(pkg_is_installed dev-db/mariadb)" ||
+		elog "You'll have to configure amarok to use an external database server:"
+		if use mariadb ; then
+			elog "    $(pkg_is_installed dev-db/mariadb)"
+			elog "    For preliminary configuration of MariaDB Server please refer to"
+			elog "    https://wiki.gentoo.org/wiki/MariaDB#Configuration"
+		else
 			elog "    $(pkg_is_installed dev-db/mysql)"
+			elog "    For preliminary configuration of MySQL Server please refer to"
+			elog "    https://wiki.gentoo.org/wiki/MySQL#Configuration"
+		fi
 		elog "Please read https://community.kde.org/Amarok/Community/MySQL for details on how"
-		elog "to configure the external db and migrate your data from the embedded database."
+		elog "to configure the external database and migrate your data from the embedded database."
+		elog "To create external amarok database with default user/password please:"
+		elog "    1. Make sure that MySQL or MariaDB is installed"
+		elog "    2. If it's required make preliminary configuration of MySQL or MariaDB"
+		elog "    3. Ensure that 'mysql' service is started and then run command 'emerge --config amarok'"
+		elog "    4. On 'Configure Amarok - Database' menu page check 'Use external MySQL database' and press OK"
```

Comment:
Maybe we should check how to enable it by default. Just a thought.

Reply:
I tried to add
```
src_prepare() {
       default
       # Switch on "Use external MySQL database" by default
       sed -i '/<group name="MySQL">/,/<default>false<\/default>/s/false/true/' src/amarokconfig.kcfg || die
       ecm_src_prepare
}
```
but it's not enough as then after initial configuration via `emerge --config amarok` and launching amarok it can't use database (I remove it before initial configuration at testing). It fail even after uncheck this option, restart and check it again. And I can't fix it for user anynore even with explicitly setup velues for MySQL in user's `amarokrc`.

While without this block it's OK: at first launch I check appropriate option, restart and database is used successfully.